x/oracle/types/asset.go

```diff
@@ -0,0 +1,48 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	appparams "github.com/CosmosContracts/juno/v12/app/params"
+)
+
+const (
+	JunoDenom       string = appparams.BondDenom
+	JunoSymbol      string = "JUNO"
+	JunoExponent           = uint32(6)
+	AtomDenom       string = "ibc/6842C591DC4588411A565C9FF650FB15A17DFE3F0A43201E8141E4D14B8D171A"
```

Comment:
This is dig channel-37 not atom

```
junod q ibc-transfer denom-trace ibc/6842C591DC4588411A565C9FF650FB15A17DFE3F0A43201E8141E4D14B8D171A
denom_trace:
  base_denom: udig
  path: transfer/channel-37
```

ATOM channel-1 is `ibc/C4CFF46FD6DE35CA4CF4CE031E643C8FDC9BA4B99AE598E9B0ED98FE3A2319F9` for juno.